### PR TITLE
pysatSpaceWeather Updates

### LINKED ISSUE #3: Update templates based on other packages
- [x] pysatCDAAC
- [x] pysatMadrigal
- [x] pysatMissions
- [x] pysatModels
- [x] pysatNASA
- [x] pysatSeasons
- [x] pysatSpaceWeather
- [x] pysat

### PR DESCRIPTION
**Pull request by aburrell**
# Description

Partially Addresses #3  by adding and adjusting templates based on pysatSpaceWeather

## Type of change

- New feature (non-breaking change which adds functionality)

# How Has This Been Tested?

Not tested

**Test Configuration**:
* Operating system: OS X Big Sur
* Version number: Python 3.9
* Any details about your local setup that are relevant

# Checklist:

- [x] Make sure you are merging into the ``develop`` (not ``main``) branch
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules
- [ ] Add a note to ``CHANGELOG.md``, summarizing the changes

If this is a release PR, replace the first item of the above checklist with the release
checklist on the wiki: https://github.com/pysat/pysat/wiki/Checklist-for-Release

### CODE REVIEW

**Comment by aburrell** [comment]
@jklenzing feel free to rebase to main and/or delete develop.

File: docs/conf.py [hunk 1]
```diff
@@ -0,0 +1,176 @@
...
+with open(os.path.join(doc_dir, "..", project, "version.txt"), "r") as fin:
+    version = fin.read().strip()
+release = '{:s}-alpha'.format(version)  # Include alpha/beta/rc tags.
+
+# The language for content autogenerated by Sphinx. Refer to documentation
+# for a list of supported languages.
+#
+# This is also used if you do content translation via gettext catalogs.
+# Usually you set "language" from the command line for these cases.
+language = None
```
Comment (jklenzing) [inline docs/conf.py]:
```suggestion
language ='en'
```
`None` is deprecated here

**Review by jklenzing — APPROVED** [review]
Looks good.  One small change based on conf.py deprecations

**Comment by jklenzing** [comment]
> @jklenzing feel free to rebase to main and/or delete develop.

OK.  dropped develop for the time being.

**Pull request merged**